packages/agoric-cli/test/test-inter-cli.js

```diff
@@ -82,13 +83,12 @@ const offerSpec1 = harden({
         numerator: mk(bslot.IST, 9n),
         denominator: mk(bslot.ATOM, 1n),
       },
-      want: mk(bslot.ATOM, 5000000n),
+      want: mk(bslot.ATOM, 5_000_000n),
     },
     proposal: {
       exit: { onDemand: null },
-      give: {
-        Currency: mk(bslot.IST, 50000000n),
-      },
+      give: { Currency: mk(bslot.IST, 50_000_000n) },
+      want: { Collateral: mk(bslot.ATOM, 5_000_000n) },
```

Comment:
Good question... @Chris-Hibbert ?

I'd love to see contract unit tests in this area.

I considered adding one but figured maybe I have already spent too much time on this bidding CLI and I should check with some reviewers about how much testing and of what sort is cost-effective here.

Reply:
It's completely legal for the `want` in the proposal to be different. The effect is that it specifies a minimum fill for the order. If we think people will want that functionality, we could add support for the auction contract to pay attention. Actually the auction wouldn't do anything different. If it couldn't completely satisfy the `want`, it would exit the seat. 

Currently it just allows Zoe to kill offers when the auction tries to reallocate less than the requested amount. Most of the time, if the `want` is reasonable, the auction will be likely to surpass it.